contents/handbook/people/team-structure/session-recording/index.mdx

```diff
@@ -9,6 +9,7 @@ hideAnchor: true
 
 - [Ben White (Team Lead, Full Stack Engineer)](/handbook/company/team/#ben-white-full-stack-engineer)
 - [Alex Kim (Full Stack Engineer)](/handbook/company/team/#alex-kim-full-stack-engineer)
+- [Emanuele Capparelli (Growth Engineer)](/handbook/company/team#ben-white-full-stack-engineer)
```

Comment:
flyby: the URL here and elsewhere is wrong for Em

Reply:
thanks! Fixed